Command/Proxy/ClearMetadataCacheDoctrineCommand.php

```diff
@@ -3,15 +3,15 @@
 namespace Doctrine\Bundle\DoctrineBundle\Command\Proxy;
 
 use Doctrine\ORM\Tools\Console\Command\ClearCache\MetadataCommand;
-use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * Command to clear the metadata cache of the various cache drivers.
  */
 class ClearMetadataCacheDoctrineCommand extends MetadataCommand
```

Comment:
Pretty much all of those commands are only changing the name and description of the proxied ORM commands now.

Whats our plan with those? Should we remove them at some point?

Reply:
For descriptions, I think we should use the upstream description (or submit the bundle descriptions upstream if they are better).
For names, we need to make sure that upstream commands allow to inject a name in the constructor (and then pass it to the parent constructor of Command which accepts a name) instead of calling `->setName` with a fixed name. Once this is OK, we won't need the proxy commands anymore.

Reply:
I think we could also change upstream command to use $defaultName property. Then we could override command name via console.command tag.

As for getting rid of these commands, definitely. They were needed at some point when upstream didn't support --em option, but that's no longer the case.

Reply:
I will create a new issue then to tackle this

Reply:
See https://github.com/doctrine/DoctrineBundle/issues/1524